.github/workflows/docker-continuous-delivery.yml

```diff
@@ -0,0 +1,90 @@
+name: Continuous Delivery
+
+on:
+  push:
+    branches: ["main", "INT"]
```

Comment:
shouldn't we also include the QA branch?

Reply:
In general, we use the `main` branch in the _QA_ environment before generating a semantic version release for _Production_